src/components/Button/README.md

```diff
@@ -1,4 +1,5 @@
 # Button
+For a simplified text-only button with no padding or focus/active/hover states, see: [TextButton](#TextButton).
```

Comment:
note: this currently does not work; it changes the link correctly in the address bar, but it doesn't actually route you to the TextButton doc. i would like to open a new issue to implement the ability to link between pages in the styleguide; it would also apply particularly to the "table of contents" idea i mentioned in #238 "2b".

lmk what you think i should do with this for now, e.g.
- leave as-is (once clicked, you can refresh to get to the TextButton page)
- remove the link but maybe bold `TextButton` instead
- open an issue to "implement the ability to link between pages in the styleguide" as mentioned above
- etc. etc.

Reply:
Change the link from `#TextButton` to `#/TextButton` and it works.

Reply:
Change the link from #TextButton to #/TextButton and it works.

```suggestion
For a simplified text-only button with no padding or focus/active/hover states, see: [TextButton](#/TextButton).
```

Reply:
i made the commit

Reply:
🙏 🙏 TIL